DrinkManagerWeb/Models/ViewModels/DrinksViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using BLL;
+using System.Collections.Generic;
+
+namespace DrinkManagerWeb.Models.ViewModels
+{
+    public class DrinksViewModel
+    {
+        public List<Drink> Drinks { get; set; }
```

Comment:
i think that this property should be of type Pagination<Drink> this will allow us to have access to pagination properties in the view.

Reply:
Changed.